pkg/hypervisor/config.go

```diff
@@ -8,6 +8,9 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/SkycoinProject/skywire-mainnet/internal/skyenv"
```

Comment:
Could you please move lines 11 and 12 below and group with line 16?

Reply:
done